### DISCUSSION TITLE
UI/UX: please use a scrollbar on the desktop mode poll screen

### DISCUSSION THREAD

**Discussion by JohannesBe** [lukevella/rallly#143] [Ideas]
For large polls it is unclear for some users that there are options on the right.
In my last poll two out of 9 persons missed all extra options to the right of the initial rendered dates.
The `+x more...` button seems to be a non-user-intuitive way of horizontal expansion.

Thanks in advance for looking into this.

Kind regards,
J

**Comment by lukevella** [comment]
Thanks for raising this issue. I wasn't sure whether this was a concern anymore but clearly there's some work that needs to be done here. This is not a bug though so moving to discussions.

**Comment by mrToshkin** [comment]
I agree with [JohannesBe](https://github.com/JohannesBe). This is not obvious for the first time that there is the button for more options, and many people came across with this issue
<img width="699" alt="image" src="https://user-images.githubusercontent.com/54904671/212643263-96e675b4-774a-4936-9114-c7d288413572.png">

**Comment by benjaoming** [comment]
Thanks for a great project!!!

I want to pitch in that this is still an issue, both in the previous series (2.x) and current series (3.x).

It's not that anything is broken. I like the arrows for pagination, I like how you can sort of see the cut-off in version 3.x, which is an improvement since this was first reported.

## 2.x screenshot

![image](https://github.com/lukevella/rallly/assets/374612/396cb522-4303-4068-8103-a1187bc4eee8)

## 3.x screenshot

![image](https://github.com/lukevella/rallly/assets/374612/d343d799-0998-4e6e-a2e6-4c393926c722)

## Potential fixes?

There are still a couple of strategies left to _improve_ the chances that the user discovers the pagination:

1) Automatically scroll to Page 2 when all options are clicked on Page 1.
2) Indicate "Page 1 of 2" on first page
3) Make a visible progress bar at the bottom that increases for each click. The user will be aware when they have filled out the entire survey, it will be 100%. But if there are several pages, they cannot reach 100% on the first page (!)

In case of 1 and 3, they also require/encourage an explicit click on each option, making the likeliness of the "?" lower for all polls, no matter if they are paginated or not.
